src/config/constants/trading-competition/easter.ts

```diff
@@ -0,0 +1,186 @@
+export enum Tiers {
+  GOLD = 'Gold',
+  SILVER = 'Silver',
+  BRONZE = 'Bronze',
+  PURPLE = 'Purple',
+  TEAL = 'Teal',
+}
```

Comment:
I suspect this will be in every competition.

Reply:
👍